src/towncrier/test/test_format.py

```diff
@@ -327,6 +331,38 @@ def test_line_wrapping_disabled(self):
         output = render_fragments(
             template,
             None,
+            "",
+            fragments,
+            definitions,
+            ["-", "~"],
+            wrap=False,
+            versiondata={"name": "MyProject", "version": "1.0", "date": "never"},
+        )
+        self.assertEqual(output, expected_output)
+
+    def test_title_format_specified(self):
+        """
+        Title format replaces default top line rendering.
+        """
+        self.maxDiff = None
+
+        fragments = {}
+
+        definitions = OrderedDict()
+
+        expected_output = u"""A custom top line
+=================
+"""  # NOQA
+
+        template = pkg_resources.resource_string(
+            "towncrier", "templates/default.rst"
+        ).decode("utf8")
+
+        fragments = split_fragments(fragments, definitions)
+        output = render_fragments(
+            template,
+            None,
+            "A custom top line",
```

Comment:
also. to make all these render_fragments easier to read, it woud be nice to  call them with named arguments for all argumetns and not only 

```suggestion
        output = render_fragments(
            template=template,
            issue_format=None,
            top_line="A custom top line",
```